src/AccessibilityHelperTrait.php

```diff
@@ -66,11 +68,12 @@ public function setReflectionClassInstance($instance, $class = null) {
  *
  * @param string $class
  * @return Object
+ * @throws Exception
```

Comment:
PHP Storm does know it from the `use` statement, but I do agree that making ide specific changes isn't productive.